datastore/grantserver/postgres.go

```diff
@@ -66,7 +66,7 @@ func (pg *Postgres) Migrate() error {
 		return err
 	}
 
-	err = m.Migrate(8)
+	err = m.Migrate(9)
```

Comment:
This is the migration version number? Can we use a variable with a meaningful name?
```
MIGRATION_VERSION = 9
m.Migrate(MIGRATION_VERSION)
```